crates/exec-wasmtime/src/runtime/mod.rs

```diff
@@ -75,8 +75,8 @@ impl Runtime {
         let mut names = vec![];
         for (fd, file) in files.iter().enumerate() {
             names.push(file.name());
-            let (file, caps): (Box<dyn WasiFile>, _) = match file {
-                File::Null(..) => (Box::new(Null), FileCaps::all()),
+            let (file, access): (Box<dyn WasiFile>, _) = match file {
+                File::Null(..) => (Box::new(Null), FileAccessMode::WRITE),
```

Comment:
why not `all()`?

Reply:
There is no `all`.

```rust
pub struct FileAccessMode : u32 {
    const READ = 0b1;
    const WRITE= 0b10;
}
```

Reply:
I guess it can be removed completely then

Reply:
caps != access